roles/servicetelemetry/templates/manifest_grafana.j2

```diff
@@ -0,0 +1,25 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: {{ meta.name }}
+  namespace: {{ meta.namespace }}
+spec:
+  ingress:
+    enabled: {{ graphing_spec_ingress_enabled }}
+  config:
+    auth:
+      disable_signout_menu: true
```

Comment:
Suggest false unless there is a good reason

Reply:
Sorry, the GH output makes it confusing. Are you referring to `graphing_spec_ingress_enabled` or `disable_signout_menu`?

If the former, I only put it to what our current default already is, but I did look at this and thought about making this inverted to our current default? What do you think? I feel it's safe enough to revert it.

Maybe I should also add another parameter to make it easier to expose?

Reply:
Sorry, `disable_signout_menu` is what I was referring to. I think it should be enabled, not disabled.

Now that you mention the ingress, though, considering that the defaults are not very secure, I think it should be off by default with an option to enable it at the top level.

Reply:
@csibbitt ok I've created a new commit that adds the changes for defaults to both false. They are not exposed as a parameter to ServiceTelemetry (at least via API). We can rely on the auto-passthrough, but long term we should not do that.

So basically, as it stands now if you know the specific knobs to turn you can easily override from the `ServiceTelemetry` object, but they are not explicitly presented via the API (CRD, specDescriptors).

Are we ok with that?

Reply:
Sounds ok to me.